src/cf-forms/src/atoms/form-alert.less

```diff
@@ -22,7 +22,6 @@
     }
 }
 
-
```

Comment:
I see you're setting max-empty-lines to 2 in the config, but here and many other places, 2nd empty lines are being removed. Did you add the rule after you saw that, but not retry the auto-fixer?

Reply:
Hm, yeah I may have run it with the default config and then added max 2 empty lines and ran the auto-fixer, but that would only truncate 3 lines down to 2, not bring 1 line up to 2. Looking at this file though, 1 empty line seems fine for that spot. Do we have other spots where it would be preferable as 2 lines? The ones like cf-pagination are above code comment "headers" but I don't mind it as 1 line. The comments themselves are 3 lines tall.

Reply:
I don't have a consistent philosophy for when it should happen, it's sort of a know-it-when-you-see-it situation.